src/frequenz/client/electricity_trading/_client.py

```diff
@@ -346,7 +347,7 @@ async def create_gridpool_order(  # pylint: disable=too-many-arguments, too-many
         try:
             response = await cast(
                 Awaitable[electricity_trading_pb2.CreateGridpoolOrderResponse],
-                self._stub.CreateGridpoolOrder(
+                self.stub.CreateGridpoolOrder(
                     electricity_trading_pb2.CreateGridpoolOrderRequest(
                         gridpool_id=gridpool_id,
                         order=order.to_pb(),
```

Comment:
As far as I understood the documentation and example, the intended way to call functions now is
```
response = await call_stub_method(self, async lambda: await self.stub.CreateGridpoolOrder(..))
```

Reply:
Nevermind that part though, apparently `call_stub_method` isn't easily usable with async channels

Reply:
for all the non streaming methods that seems to be the way to go though.

Reply:
and for streaming there is already the `GrpcStreamBroadcaster` in place.

Reply:
Not sure I get this right, but if it doesn't work yet I'd rather wait before we implement it here

Reply:
Yeah, if it doesn't work it is a bug, and probably due to this never being used with grpcio before. I'm currently busy with some other higher prio stuff, so if nobody else can have a look at it now we'll have to delay this PR a bit until I have some time to see what's wrong/missing with grpcio for using `call_stub_method`).

Reply:
**OR** we can actually not use `call_stub_method` yet, having the client integrated without it still has value, as at least we would get a unified way to constructur/connect/close clients and use them as context managers.

Reply:
Then I'd suggest to move forward without this!